backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/AuditLoggingFailuresTest.java

```diff
@@ -84,6 +85,7 @@ void init() {
         });
   }
 
+  // I tweaked the behavior to not return the original error message in order to improve security
```

Comment:
will not returning the original error message affect our ability to debug errors we do see? 🤔

Reply:
ahhh is this related to this: https://github.com/CDCgov/prime-simplereport/pull/4243/files#r971970451 ? If so, please disregard first comment

Reply:
Yea, that's why we're logging the original error separately

https://github.com/CDCgov/prime-simplereport/blob/4ea880b87358608c21bd20bd21ec5512671b9837/backend/src/main/java/gov/cdc/usds/simplereport/config/GraphQlConfig.java#L40

more details here: https://github.com/CDCgov/prime-simplereport/pull/4243/files#r971524527